action.yml

```
@@ -138,7 +142,7 @@ runs:
       # Téléchargement de la dernière version du publisher
       - name: 📥 Download IG Publisher
         shell: bash          
-        run: wget -q https://github.com/HL7/fhir-ig-publisher/releases/latest/download/publisher.jar
+        run: wget -q https://github.com/HL7/fhir-ig-publisher/releases/download/${{inputs.ig-publisher-version}}/publisher.jar
```

Ce lien ne fonctionne qu'avec les numéros de version mais pas avec latest

Exemple :
- https://github.com/HL7/fhir-ig-publisher/releases/download/latest/publisher.jar
- https://github.com/HL7/fhir-ig-publisher/releases/download/1.6.15/publisher.jar

Il faut utiliser https://github.com/HL7/fhir-ig-publisher/releases/latest/download/publisher.jar dans le cas où c'est latest


```suggestion
        run: if [ ${{inputs.ig-publisher-version}} == latest]; then wget -q https://github.com/HL7/fhir-ig-publisher/releases/latest/download/publisher.jar; else wget -q https://github.com/HL7/fhir-ig-publisher/releases/download/${{inputs.ig-publisher-version}}/publisher.jar; fi```

---

Bien vu @nriss 
Cela montre l'interet de la review